src/index.js

```diff
@@ -206,15 +225,13 @@ export function distinctState() {
 // ifAction helper
 export function ifAction(rawActions) {
   const actions = parseActions(rawActions);
-  return (action) => action.type === '@@redux/INIT' || action.type === '@@INIT'
-    || actions.indexOf(action.type) > -1;
+  return (action) => actions.indexOf(action.type) >= 0;
 }
 // /ifAction
 
 // excludeAction helper
 export function excludeAction(rawActions = []) {
   const actions = parseActions(rawActions);
-  return (action) => action.type === '@@redux/INIT' || action.type === '@@INIT'
-    || !(actions.indexOf(action.type) > -1);
+  return (action) => !(actions.indexOf(action.type) < 0);
```

Comment:
I think you broke `excludeAction` with this, I'm pretty sure it should be either just `actions.indexOf(action.type) < 0` or `!(actions.indexOf(action.type) >= 0)`